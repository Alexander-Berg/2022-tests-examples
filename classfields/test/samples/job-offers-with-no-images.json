{
  "avito_sh_profile": {
    "profile_url": "https://www.avito.ru/i126078403",
    "profile_name": "\u0413\u0440\u0443\u043f\u043f\u0430 \u043a\u043e\u043c\u043f\u0430\u043d\u0438\u0439 369",
    "profile_phone": "8925018-10-85",
    "sh_last_visited": "2021-06-24T18:02:42+0300",
    "offers": [
      {
        "listing_url": "https://www.avito.ru/kazan/vakansii/voditel_taksi_bez_zaloga_2177595223",
        "listing_id": "2177595223",
        "listing_name": "\u0412\u043e\u0434\u0438\u0442\u0435\u043b\u044c \u0442\u0430\u043a\u0441\u0438 \u0431\u0435\u0437 \u0437\u0430\u043b\u043e\u0433\u0430",
        "listing_categories": [
          "\u0412\u0441\u0435 \u043e\u0431\u044a\u044f\u0432\u043b\u0435\u043d\u0438\u044f \u0432 \u041a\u0430\u0437\u0430\u043d\u0438",
          "\u0420\u0430\u0431\u043e\u0442\u0430 \u0432 \u041a\u0430\u0437\u0430\u043d\u0438",
          "\u0412\u0430\u043a\u0430\u043d\u0441\u0438\u0438 \u0432 \u041a\u0430\u0437\u0430\u043d\u0438",
          "\u0422\u0440\u0430\u043d\u0441\u043f\u043e\u0440\u0442, \u043b\u043e\u0433\u0438\u0441\u0442\u0438\u043a\u0430",
          "\u0421\u0432\u043e\u0431\u043e\u0434\u043d\u044b\u0439 \u0433\u0440\u0430\u0444\u0438\u043a"
        ],
        "listing_price": "95000",
        "listing_description": "\u0412\u043e\u0434\u0438\u0442\u0435\u043b\u044c \u0442\u0430\u043a\u0441\u0438 \u0431\u0435\u0437 \u0437\u0430\u043b\u043e\u0433\u0430.\n\u0423\u0434\u043e\u0431\u043d\u044b\u0435 \u0433\u0440\u0430\u0444\u0438\u043a\u0438 \u0440\u0430\u0431\u043e\u0442\u044b 6/1, 2/2, 3/3, 5/2, 2/2/2, \u043f\u043e\u0434\u0440\u0430\u0431\u043e\u0442\u043a\u0430!\n\u0415\u0436\u0435\u0434\u043d\u0435\u0432\u043d\u044b\u0435 \u0432\u044b\u043f\u043b\u0430\u0442\u044b!\n\u0411\u0415\u0417 \u041f\u0415\u0420\u0412\u041e\u041d\u0410\u0427\u0410\u041b\u042c\u041d\u042b\u0425 \u0412\u0417\u041d\u041e\u0421\u041e\u0412 \u0438 \u041d\u0410\u041a\u041e\u041f\u0418\u0422\u0415\u041b\u042c\u041d\u042b\u0425 \u0414\u0415\u041f\u041e\u0417\u0418\u0422\u041e\u0412!\n\u0410\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0438 2018 \u0433\u043e\u0434\u0430.\n\u041f\u0440\u0438\u0432\u0435\u0434\u0438 \u0434\u0440\u0443\u0433\u0430 - \u043f\u043e\u043b\u0443\u0447\u0438 \u0434\u043e 5000 \u0440\u0443\u0431\u043b\u0435\u0439!!!\n\u0410\u0440\u0435\u043d\u0434\u0430 \u0432 \u0441\u0443\u0442\u043a\u0438 \u043e\u0442 1350 \u0440\u0443\u0431\u043b\u0435\u0439!\n\u0410\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0438 \u043f\u0430\u0440\u043a\u0430:\n\u25aa \u0428\u043a\u043e\u0434\u0430 \u0420\u0430\u043f\u0438\u0434 (Skoda Rapid)\n\u25aa \u0425\u0435\u043d\u0434\u0430\u0439 \u0421\u043e\u043b\u044f\u0440\u0438\u0441 (Hyundai Solaris)\n\u25aa \u0424\u043e\u043b\u044c\u0446\u0432\u0430\u0433\u0435\u043d \u041f\u043e\u043b\u043e (Volkswagen Polo)\n\u25aa \u041a\u0438\u0430 \u0420\u0438\u043e (Kia Rio)\n\u0410\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0438 2018 \u0433\u043e\u0434\u0430.\n\u260e\u0417\u0432\u043e\u043d\u0438\u0442\u0435 \u0441\u0435\u0439\u0447\u0430\u0441 \u0438 \u043f\u0440\u0438\u0445\u043e\u0434\u0438\u0442\u0435 \u043a \u043d\u0430\u043c!\n\u0423\u0441\u043b\u043e\u0432\u0438\u044f:\n\u2022 \u0413\u0438\u0431\u043a\u0438\u0439 \u0413\u0440\u0430\u0444\u0438\u043a \u0440\u0430\u0431\u043e\u0442\u044b 6/1 \u0438\u043b\u0438 7/0. \u0412\u043e\u0437\u043c\u043e\u0436\u0435\u043d \u0438\u043d\u0434\u0438\u0432\u0438\u0434\u0443\u0430\u043b\u044c\u043d\u044b\u0439 \u043f\u043e\u0434\u0431\u043e\u0440 \u0433\u0440\u0430\u0444\u0438\u043a\u0430.\n\u2022 \u0410\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u044c \u0431\u0435\u0437 \u0437\u0430\u043b\u043e\u0433\u0430 \u0438 \u0434\u0435\u043f\u043e\u0437\u0438\u0442\u0430;\n\u2022 \u0415\u0436\u0435\u0434\u043d\u0435\u0432\u043d\u044b\u0435 \u0432\u044b\u043f\u043b\u0430\u0442\u044b \u0431\u0435\u0437\u043d\u0430\u043b\u0438\u0447\u043d\u044b\u0445 \u0441\u0440\u0435\u0434\u0441\u0442\u0432 \u0441 \u0442\u0430\u043a\u0441\u043e\u043c\u0435\u0442\u0440\u0430.\n\u2022 \u0410\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0438 \u0431\u0440\u0435\u043d\u0434\u0438\u0440\u043e\u0432\u0430\u043d\u043d\u044b\u0435, \u043e\u043a\u043b\u0435\u0439\u043a\u0430 \u0431\u0440\u0435\u043d\u0434\u0430 \u042f\u043d\u0434\u0435\u043a\u0441 \u0422\u0430\u043a\u0441\u0438;\n\u2022 \u0417\u0430\u043f\u0440\u0430\u0432\u043a\u0430 \u0438 \u043c\u043e\u0439\u043a\u0430 \u0437\u0430 \u0441\u0447\u0435\u0442 \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u044f;\n\u2022 \u0420\u0435\u043c\u043e\u043d\u0442 \u0438 \u0421\u043c\u0435\u043d\u0430 \u0440\u0435\u0437\u0438\u043d\u044b \u0437\u0430 \u0441\u0447\u0435\u0442 \u0442\u0430\u043a\u0441\u043e\u043f\u0430\u0440\u043a\u0430;\n\u2022 \u041a\u0440\u0443\u0433\u043b\u043e\u0441\u0443\u0442\u043e\u0447\u043d\u0430\u044f \u0442\u0435\u0445\u043d\u0438\u0447\u0435\u0441\u043a\u0430\u044f \u043f\u043e\u0434\u0434\u0435\u0440\u0436\u043a\u0430;\n\u2022 \u0410\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0438 \u043f\u043e\u043b\u043d\u043e\u0441\u0442\u044c\u044e \u0443\u043a\u043e\u043c\u043f\u043b\u0435\u043a\u0442\u043e\u0432\u0430\u043d\u044b \u0438 \u0433\u043e\u0442\u043e\u0432\u044b \u043a \u0440\u0430\u0431\u043e\u0442\u0435, \u043f\u043e\u043b\u043d\u044b\u0439 \u043f\u0430\u043a\u0435\u0442 \u0434\u043e\u043a\u0443\u043c\u0435\u043d\u0442\u043e\u0432 (\u043f\u0443\u0442\u0435\u0432\u044b\u0435 \u043b\u0438\u0441\u0442\u044b, \u0434\u0438\u0430\u0433\u043d\u043e\u0441\u0442\u0438\u0447\u0435\u0441\u043a\u0430\u044f \u043a\u0430\u0440\u0442\u0430, \u0440\u0430\u0437\u0440\u0435\u0448\u0435\u043d\u0438\u0435)\n\u2022 \u0417\u0430\u0440\u0430\u0431\u043e\u0442\u043d\u0430\u044f \u043f\u043b\u0430\u0442\u0430 \u0434\u043e 60% \u043e\u0442 \u0432\u044b\u0440\u0443\u0447\u043a\u0438.\n\u2022 \u0420\u0435\u0433\u0443\u043b\u044f\u0440\u043d\u043e \u043f\u043e\u043f\u043e\u043b\u043d\u044f\u0435\u043c \u0442\u0430\u043a\u0441\u043e\u043f\u0430\u0440\u043a \u0441\u0432\u0435\u0436\u0438\u043c\u0438 \u0430\u0432\u0442\u043e (\u043a\u043e\u043c\u0444\u043e\u0440\u0442, \u0431\u044e\u0434\u0436\u0435\u0442\u043d\u044b\u0439, \u044d\u043a\u043e\u043d\u043e\u043c);\n\u2022 \u0412\u0441\u0435 \u0430\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0438 \u0434\u043e\u043c\u0430\u0448\u043d\u0438\u0435;\n\u2022 \u0415\u0441\u0442\u044c \u0430\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0438 \u043d\u0430 \u0433\u0430\u0437\u0443;\n\u2022 \u041f\u043e\u0434\u043a\u043b\u044e\u0447\u0430\u0435\u043c \u043a \u0437\u0430\u043a\u0430\u0437\u0430\u043c \u042f\u043d\u0434\u0435\u043a\u0441 \u0422\u0430\u043a\u0441\u0438 (Yandex Taxi);\n\u2022 \u041f\u0440\u0438\u043e\u0440\u0438\u0442\u0435\u0442 \u0432 \u0440\u0430\u0441\u043f\u0440\u0435\u0434\u0435\u043b\u0435\u043d\u0438\u0438 \u0437\u0430\u043a\u0430\u0437\u043e\u0432; \u043a\u043e\u0440\u043e\u043d\u0430 \u042f\u043d\u0434\u0435\u043a\u0441.\n\u0422\u0440\u0435\u0431\u043e\u0432\u0430\u043d\u0438\u044f \u043a \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u044e:\n\u2022 \u0412\u043e\u0434\u0438\u0442\u0435\u043b\u044c\u0441\u043a\u043e\u0435 \u0443\u0434\u043e\u0441\u0442\u043e\u0432\u0435\u0440\u0435\u043d\u0438\u0435 \u043a\u0430\u0442\u0435\u0433\u043e\u0440\u0438\u0438 \u0412\n\u2022 \u0421\u0442\u0430\u0436 \u043f\u043e \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u044c\u0441\u043a\u043e\u043c\u0443 \u0443\u0434\u043e\u0441\u0442\u043e\u0432\u0435\u0440\u0435\u043d\u0438\u044e \u0431\u043e\u043b\u0435\u0435 3-\u0445 \u043b\u0435\u0442\n\u260e\u0417\u0432\u043e\u043d\u0438\u0442\u0435 \u0441\u0435\u0439\u0447\u0430\u0441 \u0438 \u043f\u0440\u0438\u0445\u043e\u0434\u0438\u0442\u0435 \u043a \u043d\u0430\u043c!\n\u0420\u0430\u0431\u043e\u0442\u0430\u0435\u043c \u043f\u043e \u0431\u0443\u0434\u043d\u044f\u043c \u0441 9:00 \u0434\u043e 18:00.\n\u0410\u0434\u0440\u0435\u0441 \u043f\u0430\u0440\u043a\u0430: \u0433.\u041a\u0430\u0437\u0430\u043d\u044c \u0423\u043b. \u0414\u0430\u0443\u0440\u0441\u043a\u0430\u044f, 18 (\u0424\u043e\u043b\u044c\u043a\u0441\u0432\u0430\u0433\u0435\u043d \u0426\u0435\u043d\u0442\u0440), \u0432\u0445\u043e\u0434 \u0441\u043f\u0440\u0430\u0432\u0430 \u043e\u0442 \u043e\u0441\u043d\u043e\u0432\u043d\u043e\u0433\u043e \u0446\u0435\u043d\u0442\u0440\u0430\u043b\u044c\u043d\u043e\u0433\u043e \u0432\u0445\u043e\u0434\u0430, 2 \u044d\u0442\u0430\u0436, \u043a\u0430\u0431.202.\n\u0418\u0449\u0438\u0442\u0435 \u043d\u0430\u0448\u0438 \u0434\u0440\u0443\u0433\u0438\u0435 \u043f\u0440\u0435\u0434\u043b\u043e\u0436\u0435\u043d\u0438\u044f \u043f\u043e \u0440\u0430\u0431\u043e\u0442\u0435 \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u0435\u043c \u0442\u0430\u043a\u0441\u0438 \u043d\u0430 \u0430\u0432\u0438\u0442\u043e.\n\u0412\u044b \u043c\u043e\u0436\u0435\u0442\u0435 \u043e\u0431\u0440\u0430\u0449\u0430\u0442\u044c\u0441\u044f \u043a \u043d\u0430\u043c, \u0435\u0441\u043b\u0438 \u0438\u0449\u0438\u0442\u0435 \u0432\u0430\u043a\u0430\u043d\u0441\u0438\u0438:\n\u0412\u043e\u0434\u0438\u0442\u0435\u043b\u044c \u0442\u0430\u043a\u0441\u0438, \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u044c \u0442\u0430\u043a\u0441\u0438 \u043d\u0430 \u0430\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0435 \u0442\u0430\u043a\u0441\u043e\u043f\u0430\u0440\u043a\u0430, \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u044c \u043d\u0430 \u0430\u0432\u0442\u043e \u043f\u0430\u0440\u043a\u0430, \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u044c \u0442\u0430\u043a\u0441\u0438 \u043d\u0430 \u0430\u0440\u0435\u043d\u0434\u043d\u043e\u043c \u0430\u0432\u0442\u043e\u043c\u043e\u0431\u0438\u043b\u0435.\n\u0415\u0441\u043b\u0438 \u0432\u044b \u0438\u0441\u043a\u0430\u043b\u0438 \u0430\u0440\u0435\u043d\u0434\u0430 \u0430\u0432\u0442\u043e \u043f\u043e\u0434 \u0442\u0430\u043a\u0441\u0438, \u0442\u043e \u0440\u0430\u0441\u0441\u043c\u043e\u0442\u0440\u0438\u0442\u0435 \u0432\u043e\u0437\u043c\u043e\u0436\u043d\u043e\u0441\u0442\u044c \u0440\u0430\u0431\u043e\u0442\u044b \u0432\u043e\u0434\u0438\u0442\u0435\u043b\u0435\u043c \u0442\u0430\u043a\u0441\u0438 \u0432 \u043d\u0430\u0448\u0435\u0439 \u043a\u043e\u043c\u043f\u0430\u043d\u0438\u0438.\n\u0418\u0449\u0438\u0442\u0435 \u043d\u0430\u0441 \u043d\u0430 \u0410\u0432\u0438\u0442\u043e!",
        "listing_address": "\u0420\u0435\u0441\u043f\u0443\u0431\u043b\u0438\u043a\u0430 \u0422\u0430\u0442\u0430\u0440\u0441\u0442\u0430\u043d, \u041a\u0430\u0437\u0430\u043d\u044c, \u041f\u043e\u043f\u0435\u0440\u0435\u0447\u043d\u043e-\u0410\u0437\u0438\u043d\u0441\u043a\u0430\u044f \u0443\u043b., 38\u0410",
        "seller_contact": "\u041a\u0440\u0438\u0441\u0442\u0438\u043d\u0430",
        "listing_views": "152",
        "listing_date": "24-06-21",
        "seller_type": "\u0420\u0430\u0431\u043e\u0442\u043e\u0434\u0430\u0442\u0435\u043b\u044c",
        "listing_params": {
          "\u0421\u0444\u0435\u0440\u0430 \u0434\u0435\u044f\u0442\u0435\u043b\u044c\u043d\u043e\u0441\u0442\u0438:": "\u0422\u0440\u0430\u043d\u0441\u043f\u043e\u0440\u0442, \u043b\u043e\u0433\u0438\u0441\u0442\u0438\u043a\u0430",
          "\u0413\u0440\u0430\u0444\u0438\u043a \u0440\u0430\u0431\u043e\u0442\u044b:": "\u0421\u0432\u043e\u0431\u043e\u0434\u043d\u044b\u0439 \u0433\u0440\u0430\u0444\u0438\u043a",
          "\u041e\u043f\u044b\u0442 \u0440\u0430\u0431\u043e\u0442\u044b:": "\u041d\u0435 \u0438\u043c\u0435\u0435\u0442 \u0437\u043d\u0430\u0447\u0435\u043d\u0438\u044f"
        },
        "listing_city": "\u0420\u0435\u0441\u043f\u0443\u0431\u043b\u0438\u043a\u0430 \u0422\u0430\u0442\u0430\u0440\u0441\u0442\u0430\u043d",
        "seller_contact_method": "only-chat",
        "listing_phone": "8925018-10-85",
        "sh_last_visited": "2021-06-24T18:08:19+0300"
      }
    ]
  }
}