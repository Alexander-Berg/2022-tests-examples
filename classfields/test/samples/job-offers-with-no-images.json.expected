{
  "format": "AVITO_SH",
  "location": null,
  "externalId": "2177595223",
  "seller": {
    "contacts": {
      "phone": "8925018-10-85",
      "contactMethod": "ONLY_CHAT"
    },
    "locations": [
      {
        "address": {
          "address": "Республика Татарстан, Казань, Поперечно-Азинская ул., 38А"
        }
      }
    ]
  },
  "title": "Водитель такси без залога",
  "description": "Водитель такси без залога.\nУдобные графики работы 6/1, 2/2, 3/3, 5/2, 2/2/2, подработка!\nЕжедневные выплаты!\nБЕЗ ПЕРВОНАЧАЛЬНЫХ ВЗНОСОВ и НАКОПИТЕЛЬНЫХ ДЕПОЗИТОВ!\nАвтомобили 2018 года.\nПриведи друга - получи до 5000 рублей!!!\nАренда в сутки от 1350 рублей!\nАвтомобили парка:\n▪ Шкода Рапид (Skoda Rapid)\n▪ Хендай Солярис (Hyundai Solaris)\n▪ Фольцваген Поло (Volkswagen Polo)\n▪ Киа Рио (Kia Rio)\nАвтомобили 2018 года.\n☎Звоните сейчас и приходите к нам!\nУсловия:\n• Гибкий График работы 6/1 или 7/0. Возможен индивидуальный подбор графика.\n• Автомобиль без залога и депозита;\n• Ежедневные выплаты безналичных средств с таксометра.\n• Автомобили брендированные, оклейка бренда Яндекс Такси;\n• Заправка и мойка за счет водителя;\n• Ремонт и Смена резины за счет таксопарка;\n• Круглосуточная техническая поддержка;\n• Автомобили полностью укомплектованы и готовы к работе, полный пакет документов (путевые листы, диагностическая карта, разрешение)\n• Заработная плата до 60% от выручки.\n• Регулярно пополняем таксопарк свежими авто (комфорт, бюджетный, эконом);\n• Все автомобили домашние;\n• Есть автомобили на газу;\n• Подключаем к заказам Яндекс Такси (Yandex Taxi);\n• Приоритет в распределении заказов; корона Яндекс.\nТребования к водителю:\n• Водительское удостоверение категории В\n• Стаж по водительскому удостоверению более 3-х лет\n☎Звоните сейчас и приходите к нам!\nРаботаем по будням с 9:00 до 18:00.\nАдрес парка: г.Казань Ул. Даурская, 18 (Фольксваген Центр), вход справа от основного центрального входа, 2 этаж, каб.202.\nИщите наши другие предложения по работе водителем такси на авито.\nВы можете обращаться к нам, если ищите вакансии:\nВодитель такси, водитель такси на автомобиле таксопарка, водитель на авто парка, водитель такси на арендном автомобиле.\nЕсли вы искали аренда авто под такси, то рассмотрите возможность работы водителем такси в нашей компании.\nИщите нас на Авито!",
  "condition": "INAPPLICABLE",
  "category": {
    "category": "Свободный график"
  },
  "attributes": [],
  "images": [],
  "video": null,
  "price": {
    "salary": {
      "salaryRub": 95000
    }
  }
}